pytorch_to_returnn/torch/nn/utils/rnn.py

```diff
@@ -14,14 +14,14 @@ def pack_padded_sequence(input: Tensor, lengths, batch_first=False, enforce_sort
 
 
 def pack_padded_sequence_with_batch_sizes(input: Tensor, batch_sizes: Tensor, batch_first=False) -> PackedSequence:
-  return PackedSequence(FlattenBatch(batch_major=batch_first)(input), batch_sizes)
+  return PackedSequence(FlattenBatch(batch_major=False)(input), batch_sizes)
```

Comment:
You should add some comment explaining what we discussed, i.e. that `PackedSequence` always uses the `batch_major=False` flattened variant. And that `batch_first` is about the input here, which we don't need to use as we keep that information internally around anyway.

But also, here you might want to some extra check, like:
```suggestion
  assert batch_first == input.shape[0].is_batch_dim
  return PackedSequence(FlattenBatch(batch_major=False)(input), batch_sizes)
```